src/app/shared/header/header.component.scss

```diff
@@ -0,0 +1,18 @@
+ul li {
+  list-style: none;
+}
+.form-control {
+  border-top: 0;
+  border-right: 0;
```

Comment:
try to used shorthand here and it have to be border-width css property rather than border-top I guess